platform-hub-api/app/services/kubernetes/token_lookup.rb

```diff
@@ -29,18 +31,20 @@ def lookup_static_tokens(token, kind = 'user')
           )
         end
       end
-      nil
+      found.any? ? found : nil
```

Comment:
May be nicer to just return an empty array and use that to determine whether token(s) have been found or not?